about.html

```diff
@@ -17,11 +17,12 @@
                 <a class="logo" href="index.html"><img src="images/logo.png" alt="logo"> </a> 
                 <div class="nav-links">
                     <ul class="links">
-                        <li><a href="">about</a> </li>
-                        <li><a href="">program</a> </li>
-                        <li><a href="">join</a> </li>
```

Comment:
- [ OPTIONAL] The background of your nav menu page should transparent or have the same color/pattern as the top section background. Kindly change this. :wink:

- [ OPTIONAL] What about preventing the sub-background from scrolling when the navigation bar is open? 🤔  You can use an `over-flow: hidden;` CSS property to do this and give a `height: 100%`, which will prevent seeing other sections when your mobile nav menu is open and will improve its look, as shown in this image 👇🏻.



![menubad](https://user-images.githubusercontent.com/79058364/169444164-b13e8762-37a7-49d0-8f1b-d81880135d83.png)